js/main.js

```diff
@@ -320,15 +320,17 @@ jQuery(document).on('ready', function() {
     tracks = {};
 
     $.ajax({
-      url: "./data/api/schedule.json",
+      url: "./data/api/schedule.json", //use this for testing
+      //url: "https://in.pycon.org/2017/data/api/schedule.json",
```

Comment:
This line in any case is redundant in both cases

Reply:
I'll fix this with the next PR.